src/tsickle.ts

```diff
@@ -1326,11 +1326,13 @@ class Annotator extends ClosureRewriter {
     const name = getIdentifierText(iface.name);
     this.emit(`function ${name}() {}\n`);
 
+    this.emit(`\nif (false) {\n`);
```

Comment:
can you only emit this if there are any members?

Reply:
Sure.  I think it's pretty uncommon to declare an interface with no members, but this code's dying anyway so it can't hurt.